src/sidebar/components/ShareDialog/ImportAnnotations.tsx

```diff
@@ -156,28 +157,33 @@ function ImportAnnotations({
       <FileInput onFileSelected={setFile} disabled={busy} id={fileInputId} />
       {userList && (
         <>
-          <label htmlFor={userSelectId} className="block font-medium">
```

Comment:
I'm not sure if `label`'s mechanism of focusing the element its associated with via `htmlFor` will work if the element is not a native input.

Reply:
> 2. Clicking on the label no longer focuses/activates the control

I thought this would be an acceptable drawback.

> 1. When the select control is focused, the screen reader no longer reads the label after the current content

This one on the other hand, was a good thing to have.

> I'm not sure if label's mechanism of focusing the element its associated with via htmlFor will work if the element is not a native input.

`htmlFor` can be used in labels to link with labelable elements, and according to [the spec](https://html.spec.whatwg.org/multipage/forms.html#category-label), `button` is one of them, so perhaps we can in fact keep this. I'll do some tests.

We will need https://github.com/hypothesis/frontend-shared/issues/1285 to be addressed first though. Specifically the point around `SelectNext should implement HTMLButtonAttributes, and pass them down to the toggle button, excluding those attributes internally handled, like aria-haspopup and aria-controls`

Reply:
> I thought this would be an acceptable drawback.

Taking the native `<select>` we have now, [React Spectrum's Picker](https://react-spectrum.adobe.com/react-spectrum/Picker.html) and [MUI Select](https://mui.com/material-ui/react-select/) as a reference, the expected behavior is that clicking the label focuses the select, but doesn't open it. I think we should try to get that working.

Reply:
> However, clicking the label does not focus the button, instead "clicks" it, which effectively toggles the listbox in this case.

Hmm. Ideally it would be the same as with the native `<select>` which just focuses the button, but I think this is a pretty minor issue compared to not labelling the button at all.